orc8r/cloud/go/services/bootstrapper/const.go

```diff
@@ -13,4 +13,14 @@ limitations under the License.
 
 package bootstrapper
 
-const ServiceName = "BOOTSTRAPPER"
+const (
+	ServiceName  = "BOOTSTRAPPER"
+	DBTableName  = "bootstrapper"
```

Comment:
```suggestion
	BlobstoreTableName  = "bootstrapper"
```

And fix throughout -- the fact that blobstore happens to back to a db is an implementation detail